### DISCUSSION TITLE
What is this? addedMutatedNodes empty skip

### DISCUSSION THREAD

**Discussion by jcenlo** [GrapesJS/grapesjs#4703] [Q&A] [answered]
During the development of my project, I am observing these messages in my console,
<img width="495" alt="Screenshot 2022-11-01 at 14 48 12" src="https://user-images.githubusercontent.com/1485745/199249026-17045fa9-98fa-4e3e-9486-98d25cef25be.png">

Is it something that should worry me ?

i am not sure if is ok to get this messages

Thanks for the help

**Comment by artf** [comment, marked as answer]
Doesn't look like it's coming from GrapesJS
